### PR TITLE
Cherry-pick #12931 to 7.3: Update Go to 1.12.7

### PR DESCRIPTION
**Pull request by mikemadden42**
Cherry-pick of PR #12931 to 7.3 branch. Original message:

### CODE REVIEW

File: CHANGELOG.next.asciidoc [hunk 1]
```diff
@@ -14,6 +14,8 @@ https://github.com/elastic/beats/compare/v7.0.0-alpha2...master[Check the HEAD d
 - Update to Golang 1.12.4. {pull}11782[11782]
 - Update to ECS 1.0.1. {pull}12284[12284] {pull}12317[12317]
 - Default of output.kafka.metadata.full is set to false by now. This reduced the amount of metadata to be queried from a kafka cluster. {pull}12738[12738]
+- Fixed a crash under Windows when fetching processes information. {pull}12833[12833]
```
Comment (andrewkroh) [inline CHANGELOG.next.asciidoc]: This seems like it should not be included here.

Comment (mikemadden42) [reply CHANGELOG.next.asciidoc]: I've removed the unneeded changelog entry.

**Comment by mikemadden42** [comment]
jenkins retest this please

**Comment by urso** [comment]
Not sure if it's too late already, but +1.
The failing tests are unrelated.

**Comment by mikemadden42** [comment]
Go 1.2.8 and 1.2.9 have been released.  I'm closing this PR.

**Pull request closed without merging**